tools/serverpod_cli/bin/generator/class_generator_dart.dart

```diff
@@ -2,7 +2,7 @@ import 'package:code_builder/code_builder.dart';
 
 import 'class_generator.dart';
 import 'config.dart';
-import 'protocol_definition.dart';
+import 'protocol_definition.dart' hide EnumValue;
```

Comment:
It's somewhat confusing that the same name is used for two classes in different code files. It would be better if we could find more distinctive names as the code may be hard to follow otherwise.

Reply:
`EnumValueDefinition`?

Reply:
Better? 08e4109b

Reply:
Let's go with that.